master/internal/experiment/bulk_action.go

```diff
@@ -148,8 +151,15 @@ func editableExperimentIds(ctx context.Context, inputExpIDs []int32,
 		Model(&expIDs).
 		Column("e.id").
 		Join("JOIN projects p ON e.project_id = p.id").
-		Where("NOT e.archived").
-		Where("e.id IN (?)", bun.In(experimentIDList))
+		Where("NOT e.archived")
+
+	if filters == nil && len(experimentIDList) == 1 {
+		query = query.Where("e.id = ?", experimentIDList[0])
```

Comment:
Is there a reason to handle the `len(experimentIDList) == 1` case separately rather than letting it be, for example
```sql
WHERE e.id IN (12)
```
? If not, seems like it'd make this a tad more readable to have one fewer `if` block.

Reply:
This is was a quirk of the fact that our single op endpoints use the bulk_action functions under the hood, these endpoints don't require a project id to be specified. I'm not overly attached to the approach I used here. 

I understand the benefit of leveraging the bulk_action func's, but was hesitant to require the project id to be specified on the single op endpoints since IIRC our CLI leverages the bindings.

Reply:
Another alternative would be to implement the special value that had been floated expresses the action is global or non-project dependent.

Reply:
OH! Yes, I see what you mean. That project_id `Where` slipped by me and I thought this `if` block was about handling SQL `WHERE id = 12` vs `WHERE id IN (12)`.

I think a sentinel value for project id (or something like a `*int32` which can be `nil`) would be more explicit and would allow the function to behave according to expectations in more cases. For example, I'm imagining a case where somebody did something like
```go
idsWeCareAbout := filterExperiments(someFilters)
editableExperiments(ctx, req.ProjectID, idsWeCareAbout)
```
and someFilters happened to filter the set down to exactly one. They'd expect the project id filter to be respected.

Reply:
Absolutely, I think that's a much more consistent way to handle this than only respecting the project id if there's only one experiment we're looking at.